tools/github-actions/setup/action.yml

```diff
@@ -6,6 +6,10 @@ inputs:
     description: 'Enable Cache'
     required: false
     default: 'false'
+  enable-test-app-cache:
```

Comment:
Why putting this cache option in the setup action when it is actually used only by 1 workflow?
Do you need to share this task? If so maybe the cache name should be part of the parameter as well.